ui/packages/catalog-ui-search/src/main/webapp/react-component/container/map-settings/map-settings.tsx

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import * as React from 'react'
+import styled from '../../styles/styled-components'
+import MapSettingsPresentation from '../../presentation/map-settings'
+import Dropdown from '../../presentation/dropdown'
+import { hot } from 'react-hot-loader'
+const Common = require('../../../js/Common.js')
+const user = require('../../../component/singletons/user-instance.js')
+const mtgeo = require('mt-geo')
+
+const exampleLat = '14.94'
+const exampleLon = '-11.875'
+const exampleDegrees = `${mtgeo.toLat(exampleLat)} ${mtgeo.toLon(exampleLon)}`
+const exampleDecimal = `${exampleLat} ${exampleLon}`
+const exampleMgrs = '4Q FL 23009 12331'
+const exampleUtm = '14 1925mE 1513mN'
+
+const getExample = (formatValue: string) => {
+  switch (formatValue) {
+    case 'degrees':
+      return exampleDegrees
+    case 'decimal':
+      return exampleDecimal
+    case 'mgrs':
+      return exampleMgrs
+    case 'utm':
+      return exampleUtm
+  }
+  throw `Unrecognized coordinate format value [${formatValue}]`
+}
```

Comment:
I was thinking of doing that too. Do you think it would make sense to display ``Unrecognized coordinate format value [${formatValue}]`` for the example instead of throwing an error?

Reply:
I think it would probably make sense to throw an error or warning into the console and just not display the example component at all. Either way, the error handling would be better left in the place where the value is used where there is more contextual information to best decide on an error handling course of action.